resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/delegates/ClientCookie.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.resteasy.plugins.delegates;
+
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.NewCookie;
+
+/**
+ * 
+ * @author <a href="mailto:ron.sigal@jboss.com">Ron Sigal</a>
+ * @date May 20, 2016
+ */
+public class ClientCookie extends NewCookie
```

Comment:
I don't understand why you need to introduce that `ClientCookie` and can't just use `Cookie`.